axelor-account/src/main/java/com/axelor/apps/account/service/analytic/AnalyticAccountServiceImpl.java

```diff
@@ -119,14 +118,17 @@ public String getAnalyticAccountDomain(
       } else {
         domain +=
             "in ("
-                + accountConfigRepository.findByCompany(analyticDistributionTemplate.getCompany())
-                    .getAnalyticAxisByCompanyList().stream()
-                    .map(it -> it.getAnalyticAxis().getId().toString())
-                    .collect(Collectors.toList())
+                + Joiner.on(",")
+                    .join(
+                        accountConfigRepository
+                            .findByCompany(analyticDistributionTemplate.getCompany())
```

Comment:
I know it was like this before but change it and use:
`accountConfigService.getAccountConfig(company)`